dpctl/tensor/_set_functions.py

```diff
@@ -624,3 +637,138 @@ def unique_all(x: dpt.usm_ndarray) -> UniqueAllResult:
         inv,
         _counts,
     )
+
+
+def isin(
+    x: Union[dpt.usm_ndarray, int, float, complex, bool],
+    test_elements: Union[dpt.usm_ndarray, int, float, complex, bool],
+    /,
+    *,
+    invert: Optional[bool] = False,
+) -> dpt.usm_ndarray:
+    """
+    Tests `x in test_elements` for each element of `x`. Returns a boolean array
```

Comment:
It seems the function signature needs to be duplicated in the docstring comment for better formatting
```suggestion
    """isin(x, test_elements, /, *, invert=False)

    Tests `x in test_elements` for each element of `x`. Returns a boolean array
```